tests/spark_autologging/test_pyspark_ml_autologging.py

```diff
@@ -0,0 +1,229 @@
+import importlib
+import inspect
+import pytest
+from collections import namedtuple
+from unittest import mock
+
+import mlflow
+from mlflow.utils import (
+    truncate_dict,
+)
+from mlflow.utils.validation import (
+    MAX_PARAM_VAL_LENGTH,
+    MAX_ENTITY_KEY_LENGTH,
+)
+
+from tests.autologging.fixtures import test_mode_off
+from tests.spark_autologging.utils import spark_session  # pylint: disable=unused-import
+from pyspark.ml.linalg import Vectors
+from pyspark.ml.regression import LinearRegression
+from mlflow.pyspark.ml import _should_log_model, _get_instance_param_map, _log_model_allowlist, \
+    _skip_log_model_warning_msg, _skip_autolog_on_fit_with_a_list_of_params_msg
+
+MODEL_DIR = "model"
+MLFLOW_PARENT_RUN_ID = "mlflow.parentRunId"
+
+
+@pytest.fixture(scope="module")
+def dataset_binomial(spark_session):
+    yield spark_session.createDataFrame(
+        [(1.0, Vectors.dense(1.0)),
+         (0.0, Vectors.sparse(1, [], []))] * 100,
+        ["label", "features"])
+
+
+@pytest.fixture(scope="module")
+def dataset_multinomial(spark_session):
+    yield spark_session.createDataFrame(
+        [(1.0, Vectors.dense(1.0)),
+         (0.0, Vectors.sparse(1, [], [])),
```

Comment:
Just curious why we use a sparse vector here.

Reply:
Just copy from other place, no special meaning

Reply:
I see. Does `Vectors.sparse(1, [], [])` is equivalent to a missinge value?